notify/grafana_alertmanager_test.go

```diff
@@ -51,33 +50,33 @@ func TestPutAlert(t *testing.T) {
 			title: "Valid alerts with different start/end set",
 			postableAlerts: amv2.PostableAlerts{
 				{ // Start and end set.
-					Annotations: models.LabelSet{"msg": "Alert1 annotation"},
```

Comment:
This change is the only one throughout this file, I deleted a duplicated import and replaced `models` with the alias `amv2`